samples/nrf5340/smp_svr/README.rst

```diff
@@ -0,0 +1,168 @@
+
+Flashing the sample image
+*************************
+
+To upload the bundle of images and the MCUboot, we simply use ``west flash``
+like normal.
+
+For flashing the sample:
+
+.. code-block:: console
+
+    west flash
+
+Sample image: hello world!
+==========================
+
+The ``smp_svr`` app is ready to run.  Just reset your board and test the app
+with the :file:`mcumgr` command-line tool's ``echo`` functionality, which will
+send a string to the remote target device and have it echo it back:
+
+.. tabs::
+
+   .. group-tab:: Bluetooth
+
+      .. code-block:: console
+
+         sudo mcumgr --conntype ble --connstring ctlr_name=hci0,peer_name='Zephyr' echo hello
+         hello
+
+   .. group-tab:: Shell
+
+      .. code-block:: console
+
+         mcumgr --conntype serial --connstring "/dev/ttyACM2,baud=115200" echo hello
+         hello
+
+.. note::
+   The :file:`mcumgr` command-line tool requires a connection string in order
+   to identify the remote target device. In the BT sample we use a BLE-based
+   connection string, and you might need to modify it depending on the
+   BLE controller you are using.
+
+.. note::
+   In the following sections, examples will use ``<connection string>`` to represent
+   the ``--conntype <type>`` and ``--connstring=<string>`` :file:`mcumgr` parameters.
+
+
+Image upload and Image mapping to MCUboot slot
+=====================================================
+
+For this sample configuration the mcumgr supports, for upload, 3 target images, of which allows to update all parts of the firmware.
+
+The mcumgr ``image upload`` command may be provided optional ``-e -n <image>`` parameter that will
+select target image for upload; when parameter is no provided, 0 is assumed, which means "default
+behaviour", and it performs upload to the "image-1", the MCUboot secondary slot.
+
+For clarity, here is PM label to slot to ``<image>`` translation table:
+
+    +-----------------------+--------+-------------+---------------------------+
+    | PM label              | Slot   | -n <image>  |         covers            |
+    +=======================+========+=============+===========================+
+    | "mcuboot_secondary"   | slot-1 |     0       | internal application part |
+    +-----------------------+--------+-------------+---------------------------+
+    | "mcuboot_secondary_1" | slot-3 |     1       | networking                |
+    +-----------------------+--------+-------------+---------------------------+
+    | "mcuboot_secondary_2" | slot-5 |     2       | qspi application part     |
+    +-----------------------+--------+-------------+---------------------------+
+
+.. note::
+
+   The ``-e`` option actually means "no erase", and is provided to the mcumgr
+   to prevent it from sending erase command to target, before updating image.
+   The options is always needed when ``-n`` is used for image selection,
+   as the erase command is hardcoded to erase slot-1 ("image-1"),
```

Comment:
It is hardcoded to erase "secondary slot", which in case of DirectXIP execution will be the non-running/opposite slot.

Reply:
This is not DirectXIP mode.